BLSQ/iaso
# POLIO-899: add missing field to budget tab

There was a budegt tab field missing in the model and in the UI

Related JIRA tickets :  POLIO-899

## Self proofreading checklist


## Changes

- Add `re_submitted_to_orpg_operations1_at_WFEDITABLE` field in model + migration
- Add field in budget tab
- Add field in validator
- Add missing translations

## How to test

Go to polio home page, click create campaign: the field should appear
- If there's no budget: you should be able to fill  in and save 
- If there is a budget (tab is locked for edition"): if you update the value in the budget, it is saved in the tab

## Print screen / video

![Screenshot 2023-03-21 at 10 35 28](https://user-images.githubusercontent.com/38907762/226568235-fdd98692-9bf7-4aff-832a-511923b8bbda.png)
